### DISCUSSION TITLE
Ability to suppress identical files from being excluded

### DISCUSSION THREAD

**Discussion by dlockhart** [community/community#55088] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

One of my primary use cases for searching our code is to find all occurrences of a particular thing across all our repositories. Maybe it's a class, a function, an NPM dependency, a HTML element, a particular GitHub action name -- it doesn't matter, I want to find them all so I can gauge the scope of its use and potentially make a global change.

I noticed with the new code search that I was getting much fewer results than I was expecting. If I narrowed the search down to one GitHub organization, this message appeared at the bottom of the results:

> We've excluded X identical files found across repositories. `<link>`Show identical files`</link>`.

When I click "show identical files", only then did the full result set show up. Unfortunately that message doesn't appear when I search across multiple GitHub organizations (owners) -- in that case there's absolutely no indication that "25 results" is in fact not true and there are in fact more than 40.

In almost all cases, I 100% _want_ those identical files to show up in the search results. I'm counting on it.

My feedback is this:

- Either stop doing hiding results from identical files, OR
- If that's not feasible, allow me to configure my search with an option to suppress this behavior. That option should be surfaced on the query-string so I can use search shortcuts in my browser to just always pass it in.
- Additionally, when searching across organizations (across owners), the "We've excluded X identical files..." message should still appear.

**Comment by mtalcott** [comment]
Same here - this resulted in a wildly different number of search results between my local repo and Github, and burned an hour of my time investigating...

I also see that the "We've excluded X identical files" message doesn't even appear consistently within the **same repository**, so the results are truncated without any message which is extra confusing.

**Comment by LiteBrite82** [comment]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
